Version Migration Guides/V3_to_V4_ParameterMigrationGuide.md

```diff
@@ -0,0 +1,178 @@
+# LDAR-Sim V4 Parameter Migration Guide
+
+## Purpose
+
+This guide is intended to assist users users who are updating to LDAR-Sim V4 by providing instructions on how to transfer as much of their parameter files as possible to the V4 format from V3. Do note that some new LDAR-Sim behavior has been introduced in V4 and some existing behavior has been changed, thus it is impossible to completely transfer all parameter files and inputs.
```

Comment:
users users?

Reply:
Suggestion: This guide is intended to assist users who wish to update to LDAR-Sim V4 by providing instructions on transferring as much of the existing V3 parameters to V4 format. Do note that new behavior has been introduced in V4 and some existing behavior has been changed, thus, it is impossible to transfer all parameter files and inputs.